pvanalytics/features/orientation.py

```diff
@@ -3,7 +3,7 @@
 from pvanalytics.util import _fit, _group
 
 
-def _conditional_fit(day, fitfunc, freq, default=0.0, min_hours=0.0,
+def _conditional_fit(day, fitfunc, minutes, freq, default=0.0, min_hours=0.0,
                      peak_min=None):
     # If there are at least `min_hours` of data in `day` and the
```

Comment:
Please add docstring-like comments (or an actual docstring) to define the inputs